operator/bundle_helpers/README.md

```diff
@@ -2,29 +2,29 @@
 
 For hermetic builds with Konflux, we need to provide the full list of resolved dependencies in `requirements.txt`.
 The dependency source files will be prefetched with Cachi2 and made available to the container image build.
-Because GHA/upstream is running on a different Python version, it uses `requirements-upstream.txt` to mange its own dependencies.
-Follow the procedure below after any dependencies change for successful builds in Konflux and mirror those changes into the `requirements-upstream.txt`.
+Because GHA/upstream is running on a different Python version, it uses `requirements-gha.txt` to mange its own dependencies.
+Follow the procedure below after any dependencies change for successful builds in Konflux and mirror those changes into the `requirements-gha.txt`.
```

Comment:
Nitpick

> Follow the procedure below ... and mirror those changes into the `requirements-gha.txt`.

This can be read like there's 1:1 copy in `requirements-gha.txt` or there's some tooling/process, whereas no tooling exists and in practice we're expected to creatively manually maintain `requirements-gha.txt`. Could you please adjust the sentence to prevent misleading. I'd even suggest to remove the mention of `requirements-gha.txt` from here and add a small section about dealing with that file.